motor/core.py

```diff
@@ -1795,3 +1796,86 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         pass
+
+class AgnosticClientEncryption(AgnosticBase):
+    __motor_class_name__ = 'MotorClientEncryption'
+    __delegate_class__ = ClientEncryption
+
+    create_data_key = AsyncCommand(doc=create_data_key_doc)
+    encrypt = AsyncCommand()
+    decrypt = AsyncCommand()
+
+    _close = AsyncCommand(attr_name='close')
```

Comment:
Why use `_close` instead of `close`?

Reply:
I'm not sure if we want to expose `close()` as a public api here so I chose to omit it for now but I can add it back if needed.

Reply:
We should expose `close` - users who don't use the context manager will want to be able to use it.